flink/src/main/scala/org/apache/mahout/flinkbindings/FlinkEngine.scala

```diff
@@ -357,12 +357,12 @@ object FlinkEngine extends DistributedEngine {
     res.collect().head
   }
 
-  private def generateTypeInformation[K: ClassTag]: TypeInformation[K] = {
+  def generateTypeInformation[K: ClassTag]: TypeInformation[K] = {
     val tag = implicitly[ClassTag[K]]
```

Comment:
better (shorter) standard helper for this line is (I think)

    implicit val ktag = classTag[K]